docs/meetings/team/2022_11_01.md

```diff
@@ -0,0 +1,80 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 9:36am
+
+**End Time** 10:45am
+
+**Location** Discord
+
+**Date** Nov 01 2022
+
+## 👋 Attendence
+
+1. Veronica
+2. Matt
+3. Edouard
+
+***Attendence Comments***
+
+Logan Parker could not make the meeting because is having a surgery today.
+
+## 🧸 Check-In
+
+## Announcements
+
+- GitStats due Friday
+
+- Gannt Chart due Friday
+
+- No Client Meeting
+  - Email Client
+
+## Last Week (Restrospective)
+
+- All tasks were completed
+
+- Veronica did some testing with Kuk
+
+- Matt and Ed were able to research Rhubarb integration
+
+- Logan got the local env set-up
+
+## This Week  
+
+### Gantt Chart
+
+- Veronica will upload on Friday
+
+### Rhubarb Peer Testing Process
+
+1) Naz could potentially provide us with a front-end for peer testing
+
+- TASK: Discord message Nazim/Devs about access to a front-end
+
+2) Users could have access to a premium account and download files and place into directory.
+
+3) Have already in-place text and audio files in a peer testing directory that the user could select from. Avoid UI as its not a focus for this project.
+
+### Phoneme Factory
+
+- TASK: Integrate Rhubarb our Node Js environment as a 'phoneme factory'
+
+- TASK: Unit tests for user inputs (file types, empty file)
+
+- TASK: GitHub Workflow for CI
+  - Testing Rhubarb?
+
+- TASK: Rhubarb into repo due Post Reading Break
+
+- EXPLORATION:
+  - How do we create an mp4 into typescript ? (Research)
+  - How do we convert phoneme factory output into a mapped sequence of frames/video
+  
+- TASK: Upload temporary phoneme assets into a directory
+
+## Client Meeting
+
+- Matt will contact other client liasons to see if meeting is needed for this week
```

Comment:
I think a progress report meeting could be useful considering we did not have a meeting last week

Reply:
That could be useful. I think it would only be good if Naz wants to check in with our progress. I do not how much we would get from a meeting this week.